app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  has_many :comments, foreign_key: :posts_id
+  has_many :likes, foreign_key: :posts_id
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  def most_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_comment_counter
```

Comment:
- [ ] Post model should have a method that updates the posts counter for a user.
```ruby
def update_post_counter
  author.increment!(:posts_counter)
end
```